content: Add some new hails (civilian and Syndicate dummy target). (#5477)
endless-sky/endless-sky: data/human/hails.txt

		"The next event will be great!"
		"That's real entertainment!"

phrase "friendly civilian"
	word
		"I"
		"Me and my friend"
	word
		" "
	word
		"had a job"
		"worked for years"
	word
		" "
	word
		"creating pixel art"
		"creating computer games"
		"creating 3D computer models"

> 		"creating websites on the Worlds Wide Web"

Reviewer: Nice.

		"The next event will be great!"
		"That's real entertainment!"

phrase "friendly civilian"
	word
		"I"
		"Me and my friend"
	word
		" "

> 	word

Reviewer 2: There is a mix of jobs and hobbies/personal projects here, and using them interchangeably in a phrase feels off-putting to me. A couple examples:
`worked in the business of cleaning corridors in an university`
`used to spend time on farming in the dirt belt`
There's a good variety here, but the downside is that some of these possible combinations sound really odd. You could remove some of the "words" at this part (I think simply removing `used to spend time on` would work), entirely change this up to make separate phrases for hobby-based things and work-based things, or remove the the "words" later that specifically reference jobs rather than things that could be jobs or hobbies (like farming, making ketchup, etc.).

Author: Good feedback! I removed "used to spend time" and I removed "worked in the business of".

		"had a job"

> 		"used to spend time on"

Reviewer 2: ```suggestion
		"used to spend time"
```
Another thing: for some of the "word"s below, having `on` at the end of this "word" is appropriate, and sometimes it isn't:
`used to spend time on farming in the dirt belt`
I'd recommend removing it here and adding `on` to the beginning of later "word"s where it's appropriate.

> 		"worked for years on"

Reviewer 2: ```suggestion
		"worked for years"
```
Same as above.

		"worked in the business of"
	word
		" "
	word
		"creating pixel art"
		"creating computer games"
		"creating 3D computer models"

> 		"creating websites on the worlds wide web"

Reviewer 2: ```suggestion
		"creating websites on the World Wide Web"
```
While "web" or "internet" may not be capitalized today, I think "World Wide Web" still qualifies for capitalization.

		"cleaning corridors in an university"

> 		"delivering furniture to peoples houses"

Reviewer 2: ```suggestion
		"delivering furniture to people's houses"
```

		"documenting complex processes"

> 		"farming in the dirt belt"

Reviewer 2: ```suggestion
		"farming in the Dirt Belt"
```
_Is_ Dirt Belt supposed to be capitalized? I actually don't know, but I think it is. Feel free to resolve if I'm wrong.

		"installing domotics into rich customers houses"

> 		"making large amounts of ketchap"

Reviewer 2: ```suggestion
		"making large amounts of ketchup"
```

		"moving furniture from warehouses to shopping centers"
		"painting car tires black"

> 		"packing fastfood into boxes"

Reviewer 2: ```suggestion
		"packing fast food into boxes"
```

		"setting up software integration processes"
		"spreading news to hundreds of people"

> 		"teaching mathematics to highschool kids"

Reviewer 2: ```suggestion
		"teaching mathematics to high school kids"
```

> 		"tending bars"

Reviewer 2: ```suggestion
		"bartending"
```

		"testing computer fabrication machines"
		"writing firmware for embedded computerchips"
	word

> 		" and look what a nice ship I'm flying now."

Reviewer 2: ```suggestion
		", and look what a nice ship I'm flying now."
```
Two independent clauses, so needs a comma.

> 		" and now I am retired and flying spaceships for fun."

Reviewer 2: ```suggestion
		", and now I'm retired and flying spaceships for fun."
```
Same as above. Thought `I'm` sounded more natural.
